packages/test-generator/integration-test-templates/cypress/e2e/form/BidirectionalOwner-Dog-spec.cy.ts

```diff
@@ -0,0 +1,31 @@
+/*
+  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+
+import { removeArrayItem } from '../../utils/form';
```

Comment:
Regarding the spec and the jsx files being named "BidirectionalOwner-Dog": It seems that your plan is to add all the 4 tests into these files.
Can I ask that they actually be separated by models? (you can see tests for CompositeDog, CompositeOwner, etc.) for reference.
While working on adding e2e's, I found that it starts getting difficult to reason about when there's multiple but related models in 1 jsx.

Reply:
Yeah that makes sense. Ill update it in next PR with the other Owner tests